test/interpreter_functional/plugins/kbn_tp_run_pipeline/server/plugin.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { CoreSetup, Plugin, HttpResponsePayload } from '../../../../../src/core/server';
+import { PluginStart as DataPluginStart } from '../../../../../src/plugins/data/server';
+import { ExpressionsServerStart } from '../../../../../src/plugins/expressions/server';
+
+export interface TestStartDeps {
+  data: DataPluginStart;
+  expressions: ExpressionsServerStart;
+}
+
+export class TestPlugin implements Plugin<TestPluginSetup, TestPluginStart, {}, TestStartDeps> {
+  public setup(core: CoreSetup<TestStartDeps>) {
+    const router = core.http.createRouter();
+
+    router.post(
+      {
+        path: '/api/interpreter_functional/run_expression',
```

Comment:
All this route does is take an expression & optional input, call `expressions.run` from the server contract, and return the result.

This way we can gradually build out a suite that tests the same expressions on both client & server.